src/log.js

```diff
@@ -360,6 +323,63 @@ export class Log {
     return error;
   }
 
+  /**
+   * @param {!Error} error
+   * @private
+   */
+  prepareError_(error) {
+    error = duplicateErrorIfNecessary(error);
+
+    if (this.suffix_) {
+      if (!error.message) {
+        error.message = this.suffix_;
+      } else if (error.message.indexOf(this.suffix_) == -1) {
+        error.message += this.suffix_;
+      }
+    } else if (isUserErrorMessage(error.message)) {
+      error.message = stripUserError(error.message);
+    }
+  }
+
+  /**
+   * @param {!Array} args
+   * @return {!Array}
+   * @private
+   */
+  maybeExpandMessageArgs_(args) {
+    return isArray(args[0])
+      ? this.expandMessageArgs_(/** @type {!Array} */ (args[0]))
+      : args;
+  }
+
+  /**
+   * Either redirects a pair of (errorId, ...args) to a URL where the full
+   * message is displayed, or displays it from a fetched table.
+   *
+   * This method is used by the output of the `transform-log-methods` babel
+   * plugin. It should not be used directly. Use the (*error|assert*|info|warn)
+   * methods instead.
+   *
+   * @param {!Array} parts
+   * @return {!Array}
+   * @private
+   */
+  expandMessageArgs_(parts) {
+    // First value should exist.
+    const id = parts.shift();
+    // Best effort fetch of message template table.
+    // Since this is async, the first few logs might be indirected to a URL even
+    // if in development mode. Message table is ~small so this should be a short
+    // gap.
+    if (getMode(this.win).development) {
+      this.fetchExternalMessagesOnce_();
+    }
+
+    return this.messages_?.[id]
+      ? [this.messages_[id]].concat(parts)
+      : [`More info at ${externalMessageUrl(id, parts)}`];
```

Comment:
FMI: do you know if this code is actually active? I didn't realize we even had started on this feature

Reply:
It has never been used/shipped, but was working (though untested) at one point. The transforms likely need updating since much of assertion code has been moved.